src/commonTest/kotlin/kotools/types/collection/NotEmptyMapTest.kt

```diff
@@ -62,6 +124,34 @@ class NotEmptyMapTest {
         )
     }
 
+    @Test
+    fun equals_should_pass_with_the_same_NotEmptyMap() {
+        val x: NotEmptyMap<Char, Int> = notEmptyMapOf('a' to 1, 'b' to 2)
+        val y: NotEmptyMap<Char, Int> = x
+        assertEquals(x, y)
+    }
```

Comment:
Use the assertions available in the [`kotools.types.Assertions.kt`](https://github.com/kotools/types/blob/5d1dc8f98f2220e96081381809cde9d261d04f8f/src/commonTest/kotlin/kotools/types/Assertions.kt) file.

```suggestion
    @Test
    fun equals_should_pass_with_the_same_NotEmptyMap() {
        val x: NotEmptyMap<Char, Int> = notEmptyMapOf('a' to 1, 'b' to 2)
        val y: NotEmptyMap<Char, Int> = x
        x shouldEqual y
    }
```